config/public-api-v1.yml

```diff
@@ -2,7 +2,7 @@
 openapi: 3.0.0
 info:
   version: 1.0.0
-  title: Apply API
+  title: Postgraduate Teacher Training Courses API
```

Comment:
```suggestion
  title: Postgraduate Teacher Training API
```

Reply:
I'm not quite sure about this. Our service is "Publish teacher training courses" and "Find teacher training courses", so I thought we should stick to that data. OTOH, we are likely going to have to expose provider data. 🤷‍♂

Reply:
I'm going to see what PaulG and Izzy think of this. ;)